google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -96,7 +98,8 @@
           new BasicNameValuePair(IS_NON_INTERACTIVE_KEY, STRING_FALSE_VALUE),
           new BasicNameValuePair(
               UNIQUE_CLIENT_ID_KEY,
-              UpdateChecker.getInstallationUID(PropertiesComponent.getInstance())));
+              UpdateChecker.getInstallationUID(PropertiesComponent.getInstance())),
```

Comment:
I may just not understand how this works, but I notice the already existing "dp" value is added in the sendEvent method and guarded by isTrackingEnabled, but for this new "dh" value, you are adding it on class init. Should we update them to make them consistent?